<h3>
    Question 1
</h3>

<img src="./diagrams/ytti.png">

In [2]:
ms<-c(95,84,81,73,61,92,88,77,76,64)

In [1]:
fg<-c(84,95,72,65,65,86,79,80,81,68)

In [3]:
summary(lm(fg~ms))


Call:
lm(formula = fg ~ ms)

Residuals:
    Min      1Q  Median      3Q     Max 
-8.5613 -4.1266 -0.3212  2.9544 14.3361 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)  
(Intercept)  26.4261    16.8129   1.572   0.1546  
ms            0.6457     0.2107   3.065   0.0155 *
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 7.09 on 8 degrees of freedom
Multiple R-squared:  0.5401,	Adjusted R-squared:  0.4826 
F-statistic: 9.395 on 1 and 8 DF,  p-value: 0.01546


Find the equation of the regression line. <font color="blue">4 points</font>

$\hat{y}=0.6457x+26.4261$

What is the linear correlation coefficient?  <font color="blue">3 points</font>

In [4]:
cor(ms,fg)

[1] 0.7349176

What are the results of a hypothesis test whether there is a linear correlation between the two variables with a significance level of 0.05? Use the table of critical values for the Pearson Correlation Coefficient. <font color="blue">2 points</font>

The Pearson correlation coefficient for $n=10$ and $\alpha=0.05$ is $0.632$.

Use the equation to predict the statistics final grade for a student who scored 70 on the mathematics aptitude test. <font color="blue">3 points</font>

The predicted value is $\hat{y}=71.6251$.

<h3>
    Question 2
</h3>

98 out of 338 members of parliament in Canada are women.

(a) Consider this problem "with replacement," even though technically it should be "without replacement." The transit committee of the Canadian parliament consists of 15 members. Assume that they are randomly drawn from the 338 members of parliament. What is the probability that strictly more than 5 committee members are female? You do not need to calculate the number, but you must give clear instructions how to find this number either on a calculator or a computer. <font color="blue">3 points</font>


In [7]:
1-pbinom(5,15,98/338)

[1] 0.2493537

(b) If any random two members of parliament speak to each other, what is the probability that they are both male; and what is the probability that they are both female? You may only use the calculator for basic arithmetic. Show your work. <font color="blue">3 points</font>

Correct:

In [2]:
((338-98)/338)*((337-98)/337)

[1] 0.5035731

In [3]:
(98/338)*(97/337)

[1] 0.08345478

Incorrect:

In [4]:
((338-98)/338)^2

[1] 0.504184

In [5]:
(98/338)^2

[1] 0.08406568

c) When a member of parliament vacates their seat between two federal elections, a by-election is called. In the last 29,808 days (since the election in 1935, when by-election rules were changed significantly), there have been 284 by-elections. Consider by-elections to be distributed such that they occur at a known average rate of 3.48 per year and independently of the time since the last by-election. Answer the following questions. <font color="blue">3 points</font>

For these questions,  you do not need to calculate the number, but you must give clear instructions how to find this number either on a calculator or a computer.

What is the probability that either no or one by-election will be held in the next year?

In [6]:
ppois(1,3.48)

[1] 0.1380172

What is the probability that there will be nine or more by-elections in the next two years?

In [8]:
1-ppois(8,3.48*2)

[1] 0.2657087

<h3>
    Question 3
</h3>

Let $C_1$ be a country in Africa and $C_2$ be a country in South America. Consider the question whether the respective proportions of babies born with a low birth weight are equal. 

(a) State the null hypothesis and the alternative hypothesis. <font color="blue">2 points</font>

$$H_{0}: p_{1}=p_{2}\qquad{}H_{a}: p_{1}\neq{}p_{2}$$

(b) If respective random samples of sizes $n_1=900$ and $n_2=700$ yielded $y_1=135$ and $y_2=77$ babies with a low birth weight, what is your conclusion at a significance level of $\alpha=0.05$? You may use either the classical or the $p$-value approach. Make sure to show all the steps of the procedure. <font color="blue">7 points</font>

In [10]:
prop.test(x=c(135,77),n=c(900,700),correct=FALSE)


	2-sample test for equality of proportions without continuity
	correction

data:  c(135, 77) out of c(900, 700)
X-squared = 5.4809, df = 1, p-value = 0.01922
alternative hypothesis: two.sided
95 percent confidence interval:
 0.007114409 0.072885591
sample estimates:
prop 1 prop 2 
  0.15   0.11 


In [12]:
pbar<-(135+77)/(900+700)

In [13]:
qbar<-1-pbar

In [15]:
z<-((135/900)-(77/700))/((((pbar*qbar)/900)+((pbar*qbar)/700))^.5);print(z)

[1] 2.341141


In [17]:
2*(1-pnorm(z))

[1] 0.0192249

<img src="./diagrams/tngtfclz.png">

<h3>
    Question 4
</h3>

<img src="./diagrams/kyavrpsq.png">

(a) What is the confidence interval for the weight of M&M bags based on the sample and a confidence level of 95%? <font color="blue">4 points</font>

In [19]:
install.packages("Rmisc")

Installing package into ‘/home/jupyter/R/x86_64-pc-linux-gnu-library/3.6’
(as ‘lib’ is unspecified)


In [20]:
library(Rmisc)

Loading required package: lattice
Loading required package: plyr


In [21]:
mm<-c(50.4,47.98,51.68,48.45,51.71,49.8,48.28,49.16,48.72,46.94,47.61,49.4,47.67,48.33,47.7,48.95,46.22,49.8,51.53,49.79,50.97,50.23,48.98,52.06,50.01,46.72,48.49,49.69,50.43,48.74)

In [30]:
CI(mm,ci=0.95)

upper     mean    lower 
49.78297 49.21467 48.64636

In [23]:
E<-49.7829726627081-49.2146666666667;print(E)

[1] 0.568306


(b) A massive study of these weights concludes that the population mean of the data is 49.2 and the population variance 2.3. Which weight in grams is exceeded by 30% of M&M bags in the population? <font color="blue">2 points</font>

In [40]:
qnorm(0.7,49.2,2.3^.5)

[1] 49.99529

(c) Using the same population mean and variance as in (b), what percentage of M&M bags weighs less than the advertised 47.9 grams? <font color="blue">3 points</font>

In [41]:
pnorm(47.9,49.2,2.3^.5)

[1] 0.1956687

<img src="./diagrams/iboupcqz.png">

<h3>
    Question 5
</h3>

A computer magazine wants to know how many of its readers use a text editor called Emacs (this final exam was written using Emacs). Google Trends gives the computer magazine a first estimate of 30.7% among computer programmers. Using this estimate of the population proportion, how many of its readers should the computer magazine survey in order to be 95% confident that the sample percentage is in error by no more than two percentage points? <font color="blue">5 points</font>

In [27]:
n<-((qnorm(0.975))^2*0.307*(1-0.307))/(0.02^2);print(n)

[1] 2043.186


<img src="./diagrams/jfixdliq.png">

<h3>
    Question 6
</h3>

<img src="./diagrams/tieasohe.png">

Under the assumption that the cost is normally distributed, what is Gavindeep's confidence interval for the population mean at a confidence level of 0.95? <font color="blue">9 points</font>

In [29]:
tc<-c(120.56,161.18,121.35,148.20,160.63,107.30,80.22,139.08,174.22,136.51,108.58,79.88,164.49,96.06,143.42,171.91,117.96,159.94,65.61,164.47,72.40,92.09)

In [32]:
CI(tc,ci=0.95)

upper     mean    lower 
142.1225 126.6391 111.1557

In [33]:
E<-142.122508498624-126.639090909091;print(E)

[1] 15.48342


<img src="./diagrams/khdaymhj.png">

<h3>
    Question 7
</h3>

<img src="./diagrams/tmbiqvoz.png">

The sample mean is 894,441.2. The sample standard deviation is 159,185.6.

(a) State the null hypothesis and the alternative hypothesis. <font color="blue">2 points</font>

$$H_{0}:\mu=818700\qquad{}H_{a}:\mu>818700$$

(b) Find the test statistic and explain how to find the associated p-value using software. <font color="blue">5 points</font>

In [34]:
wm<-c(765100,763900,1113400,1003700,995100,1029300,1050200,904400,672900,872000,754300,695000,763000,795900,1192500,1046300,788500)

In [35]:
t.test(wm,mu=818700,alternative="greater")


	One Sample t-test

data:  wm
t = 1.9618, df = 16, p-value = 0.03371
alternative hypothesis: true mean is greater than 818700
95 percent confidence interval:
 827035.8      Inf
sample estimates:
mean of x 
 894441.2 


In [37]:
length(wm)

[1] 17

(c) What is your recommendation for the real estate agent with respect to the null hypothesis, given a significance level of 0.05? <font color="blue">2 points</font>

In [39]:
qt(0.95,16)

[1] 1.745884

<img src="./diagrams/zhpyekfu.png">

<h3>
    Question 8
</h3>

<img src="./diagrams/tzsplqwd.png">

<img src="./diagrams/obbkrpxg.png">

<h3>
    Question 9
</h3>

<img src="./diagrams/cofkxcsz.png">

In [45]:
qnorm(pnorm(0,-4.3,2.5)-0.5,-4.3,2.5)

[1] -4.568198